### DISCUSSION TITLE
How do you pronounce mitmproxy?

### DISCUSSION THREAD

**Discussion by kakawait** [mitmproxy/mitmproxy#5813] [Q&A] [answered]
I know mitm is for man in the middle. Thus do you say:

- "man in the middle proxy"
- m i t m (spelling letter) proxy
- mit m (a bit like memcached with memcache then d) proxy
- other ?

Is there official way?

**Comment by mhils** [comment, marked as answer]
Within the core team we usually say "midemproxy" as one single word because that's fast and easy. :) You can hear me say it [in this very old presentation](https://youtu.be/hPuOtJbxJT8) at 0:15.

Also, while "mitm" historically has been an abbreviation for "man in the middle", the "mitm" in mitmproxy stands for "mitmproxy in the middle". We find this version to be more fun with the nice side effect of it being more inclusive. :-)
